boot/zephyr/hooks_sample.c

```diff
@@ -0,0 +1,90 @@
+/*
```

Comment:
Is this code built and executed by any samples? If so should that be included in this commit? Or is the sample in another repo?

Reply:
I see now that you are linking to a different PR in the PR description which uses this file, would it be possible to include those changes in this PR as well? To me it makes little sense to commit a file which is not included in any build.

Reply:
I added test-build in sample.yaml